qmt/__init__.py

```diff
@@ -5,5 +5,13 @@
 
 del _version
 
+import os
+import sys
+
+# make FreeCAD importable if installed via conda and when using an environment
+# remove when https://github.com/conda-forge/freecad-feedstock/issues/21) is fixed
+if "CONDA_PREFIX" in os.environ:
+    sys.path.append(os.path.join(os.environ["CONDA_PREFIX"], "lib"))
```

Comment:
this will have the effect of making everything in the `miniconda/lib` folder available right? In this wiki post https://github.com/Microsoft/qmt/wiki/Conda-Setup I recommend the more cautious approach of only making the `lib` folder in freecad available. Are the any concerns about making everything available through the path?

Also this would append to the path on every qmt import. Maybe we should check if it's already in the path?